deploy/osd-sts-ack/4.9/config.yaml

```diff
@@ -0,0 +1,9 @@
+deploymentMode: SelectorSyncSet
+selectorSyncSet:
+  matchExpressions:
+  - key: hive.openshift.io/version-major/minor
+    operator: In
+    values: ["4.8"]
+  - key: api.openshift.com/sts-4.9-upgradeable
```

Comment:
Are we changing this key every new version?

Reply:
No, as the migration is only 4.8->4.9 AFAICT

Reply:
No, there will potentially be STS changes with each minor version

Reply:
solved based on your suggestion